recipes/hipscat/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "hipscat" %}
+{% set version = "0.2.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/astronomy-commons/hipscat/archive/v{{ version }}.tar.gz
+  sha256: fe38197e62c49cfbf89115bf93a3b07ae99860bb24f4627458fde4d88a518172
+# curl -L https://github.com/astronomy-commons/hipscat/archive/v0.2.1.tar.gz | openssl sha256
```

Comment:
This commented curl line could be deleted.

Reply:
Good catch, fixed.